frontend/src/features/admin/access/components/Details.tsx

```diff
@@ -84,7 +84,7 @@ export const AccessRequestDetails: React.FC<IProps> = ({ request, onClose }) =>
         </Modal.Body>
 
         <Modal.Footer>
-          <Button variant="warning" onClick={onClose}>
+          <Button variant="primary" onClick={onClose}>
```

Comment:
Don't change this, but would be nice to extract and create a shared Close button component.

Reply:
So leave it as 'primary' or change it back to 'warning'? components are a bit outside of my realm of understanding at the moment